tests/measureSources.py

```diff
@@ -294,6 +302,31 @@ def testPixelFlags(self):
             source,
             exp,
         )
+        # Test that if there is no center and centroider that the object should look at the footprint
+        plugin, cat = makePluginAndCat(measBase.PixelFlagsAlgorithm, "test", control)
```

Comment:
My suggestion wasn't that they should be separate tests, but that a bit of refactoring within this method could make it shorter and less repetitive (probably by defining a local function and calling that). I agree that it's not worth spending time thinking about though; what you have is fine.